bids2nda/main.py

```diff
@@ -162,11 +162,22 @@ def run(args):
         dict_append(image03_dict, 'image_modality', "MRI")
         dict_append(image03_dict, 'scanner_manufacturer_pd', metadata.get("Manufacturer", ""))
         dict_append(image03_dict, 'scanner_type_pd', metadata.get("ManufacturersModelName", ""))
-        dict_append(image03_dict, 'scanner_software_versions_pd', metadata.get("HardcopyDeviceSoftwareVersion", ""))
+        dict_append(image03_dict, 'scanner_software_versions_pd', metadata.get("SoftwareVersions", ""))
         dict_append(image03_dict, 'magnetic_field_strength', metadata.get("MagneticFieldStrength", ""))
         dict_append(image03_dict, 'mri_echo_time_pd', metadata.get("EchoTime", ""))
         dict_append(image03_dict, 'flip_angle', metadata.get("FlipAngle", ""))
         dict_append(image03_dict, 'receive_coil', metadata.get("ReceiveCoilName", ""))
+        dict_append(image03_dict, 'image_slice_thickness', metadata.get("SliceThickness", ""))
+        dict_append(image03_dict, 'photomet_interpret', metadata.get("PhotometricInterpretation", ""))
+
+        plane = metadata.get("ImageOrientationPatient")
+        plane=round(plane)
+        if plane[0] == 1:
+            dict_append(image03_dict, 'image_orientation.', "Sagittal")
+        elif plane[1] == 1:
+            dict_append(image03_dict, 'image_orientation.', "Coronal")
+        elif plane[2] == 1:
+            dict_append(image03_dict, 'image_orientation.', "Axial")
```

Comment:
ha ha ;) I wondered if that is what you guessed or was informed logic (I didn't look into what they exactly mean yet)? ;)
If "logic" is informed/correct, then it might as well be just 
```python
dict_append(
    image03_dict, 'image_orientation.',
    ['Sagital', 'Coronal', 'Axial'][np.argmax(plane[:3])]
)
```
i.e. simply go after maximal one.  Not sure if it could happen that either two of them `>= 0.5` resulting to rounding to 1 in more than one, or none (all round to 0). Going after maximum would remove the ambiguity and make code shorter/without duplication.

Reply:
Also not sure if the sign couldn't be negative -- then may be also first wrap it with `np.abs()` (I think it shouldn't alter the logic of defining the slicing)

Reply:
e.g. in your case:
```
sub-sid000447/ses-1/func/sub-sid000447_ses-1_task-selfvalence_run-03_bold.json:      "ImageOrientationPatient": [0.99910127013954, -0.0366068325451, -0.0213680091858, 0.03960414059805, 0.98585875034027, 0.16283131279032],
sub-sid000448/ses-1/anat/sub-sid000448_ses-1_acq-MPRAGE_T1w.json:      "ImageOrientationPatient": [-0.004293648487, 0.95983065031276, 0.28054712135906, 0.05798604985148, 0.28031661899416, -0.9581545862429],
sub-sid000448/ses-1/dwi/sub-sid000448_ses-1_acq-DTIXhardi64_dwi.json:      "ImageOrientationPatient": [0.9983085405984, -0.0120986643356, 0.05686545603079, 0.00679536266945, 0.99568534650845, 0.09254465837924],
```
so -- was T1 coronal and the diffusion and bold Sagital? (I would think they were Axial, but I am not familiar with your protocols)  Is it just a matter of swapping Axial and Sagital? ;)